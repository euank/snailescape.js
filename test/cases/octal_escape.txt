\041\10\7\40\040\61\062
