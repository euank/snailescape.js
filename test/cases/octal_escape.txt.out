["!\b\u0007  12"]
